src/context/transaction/inputValidations.js

```diff
@@ -13,9 +13,18 @@ const isPositiveNumber = (input) => {
   }
   return input > 0;
 };
-// Validate Transaction amount is two decimals max
 
-// Validate Transaction fee is two decimals max
+// Validate number is two decimals max
+const isNumberWithMaxTwoDecimals = (input) => {
+  if (typeof input !== "number" || isNaN(input)) {
+    return false;
+  }
+  const inputString = input.toString();
+  // const [whole, decimal] = inputString.split(".");
```

Comment:
can remove commented out code

Reply:
[aaaaaaaand it's gone](https://user-images.githubusercontent.com/32176471/89136983-e578cb00-d4fb-11ea-9b94-f01c0078ba03.png)